diff.go

```diff
@@ -275,6 +275,20 @@ func (sd *SchemaDiff) FilteredTableDiffs(onlyTypes ...DiffType) []*TableDiff {
 	return result
 }
 
+// FilteredViewDiffs returns any ViewDiffs of the specified type(s).
+func (sd *SchemaDiff) FilteredViewDiffs(onlyTypes ...DiffType) []*ViewDiff {
+	result := make([]*ViewDiff, 0, len(sd.ViewDiffs))
```

Comment:
Since you are doing filtering, it probably doesn't make as much sense to preallocate capacity here. I would probably `var result []*ViewDiff` and let Go allocate memory as necessary via append. This would save some memory at the cost of potentially having to allocate multiple times. It just depends which case is more likely.